io.github.streetpea.chiaki4deck.yaml

```diff
@@ -0,0 +1,130 @@
+app-id: io.github.streetpea.Chiaki4deck
+branch: chiaki4deck
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: chiaki
+rename-icon: chiaki4deck
+rename-desktop-file: chiaki4deck.desktop
+separate-locales: false
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --allow=bluetooth
+  - --socket=pulseaudio
+  - --own-name=org.kde.*
```

Comment:
do you really need this d-bus permission?

Reply:
yes, it needs to access connected devices (i.e., controllers via bluetooth and usb)

Reply:
by owning a DBUS name? (Line 18 is the subject)